contracts/tests/test_channel_close.py

```diff
@@ -40,7 +41,12 @@ def test_close_call(get_accounts, uraiden_instance, token_instance, get_channel)
     (sender, receiver, open_block_number) = channel
     balance = channel_deposit - 10
 
-    balance_message_hash = balance_proof_hash(receiver, open_block_number, balance)
+    balance_message_hash = balance_proof_hash(
+        receiver,
+        open_block_number,
+        balance,
+        uraiden_instance.address
```

Comment:
general naming question. I see that `uraiden_instance` as seen [here](https://github.com/raiden-network/microraiden/blob/master/contracts/tests/test_uraiden.py#L69) and in the [fixture](https://github.com/raiden-network/microraiden/blob/master/contracts/tests/fixtures_uraiden.py#L54) is actually the `uraiden_contract` and not an instance of the uraiden_app as the name may imply. It's used quite a lot in tests so this should not be an urgent refactor but I would call it `uraiden_contract` and not just instance as it can be confusing.

Reply:
I use
`_instance` as an off-the-shelf deployed contract.
`_contract` as a function that can take other parameters (token, txn data)
`get_*_contract` for a lower level setup

Agree that the name is not great. Yes, will leave it like this for now, but noted.